packages/shared/src/hooks/feed/useFeedName.ts

```diff
@@ -0,0 +1,34 @@
+import { MainFeedPage } from '../../components/utilities';
+import { useFeature } from '../../components/GrowthBookProvider';
+import { Features } from '../../lib/featureManagement';
+import { SearchExperiment } from '../../lib/featureValues';
+
+interface UseFeedNameProps {
+  isSearchOn: boolean;
+  feedName: MainFeedPage;
+}
+
+interface UseFeedName {
+  isUpvoted: boolean;
+  isSortableFeed: boolean;
+}
+
+const sortableFeeds = [MainFeedPage.Popular, MainFeedPage.MyFeed];
+
+export const useFeedName = ({
```

Comment:
Just wondering if name should be different, but not blocking. 

I would expect the hook called `useFeedName` to return feed name based on parameters vs returning parameters based on feed name.